Stocks/plugin.py

```diff
@@ -100,9 +95,23 @@ def stock(self, irc, msg, args, symbol):
             change_percent=change_percent,
         )
 
-        # Print results to channel
-        irc.reply(message)
+        return message
+
+    @wrap([many('something')])
+    def stock(self, irc, msg, args, symbols):
+        """<symbol> [<symbol>, <symbol>, ...]
+
+        Returns stock data for single or multiple symbols"""
+
+        max_symbols = self.registryValue('alphavantage.maxsymbols')
+        count_symbols = len(symbols)
+
+        if count_symbols > max_symbols:
+            irc.error("Too many symbols. Maximum count {}. Your count: {}".format(max_symbols, count_symbols), Raise=True)
+
+        messages = map(lambda symbol: self.get_message(irc, symbol), symbols)
 
+        irc.reply(' | '.join(messages))
```

Comment:
Use `irc.replies(messages)` instead; it uses a configurable joiner or multiple messages instead.